website/src/contribute/get-help.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: section
+title: "Get Help Contributing"
+permalink: /contribute/get-help/
+section_menu: section-menu/contribute.html
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Get Help Contributing
+
+If you have any trouble contributing, don't give up!
+
+  - Check the [FAQ on our developers' wiki ![External link to Beam developers' Wiki]({{ site.baseurl }}/images/external-link-icon.png){:width="14px" :height="14px"}](https://cwiki.apache.org/confluence/display/BEAM/Development+Environment+FAQ){:target="_blank"}.
+  - Ask the Beam developer community on [dev@beam.apache.org]({{ site.baseurl }}/community/contact-us)
+    or [#beam on Slack]({{ site.baseurl }}/community/contact-us).
+  - The [PMC and Committer roster ![External link to Beam roster]({{ site.baseurl }}/images/external-link-icon.png){:width="14px" :height="14px"}](https://projects.apache.org/committee.html?beam){:target="_blank"} is a good list of very experienced Beam developers who also
```

Comment:
We should not encourage contacting individual folks. If this is regarding finding reviewers, then we can just refer to the owners file? Everything else should go to mailing list.

Reply:
Do you recall why "PMC & Committers" was added to the left nav? I was trying to preserve the purpose behind that. Notably, it had no context so I had no good idea why it was there.

Reply:
Don't recall. Just remember that we originally had a team link somewhere.

Reply:
Yea, it is under "Community" so I figured someone was looking for that info under "Contribute" and that is why it got added.